src/scicookie/{{cookiecutter.project_slug}}/docs-quarto/_quarto.yml

```diff
@@ -17,7 +17,15 @@ website:
 
 format:
   html:
+  {%- if cookiecutter.quarto_theme == 'Default' %}
```

Comment:
maybe would replace all this lines to:

```yaml
  {%- if cookiecutter.quarto_theme %}
    theme: {{ cookiecutter.quarto_theme.lower() }}
  {%- endif %}
```

or maybe `  {%- if cookiecutter.quarto_theme != '' %} `


but anyway .. i think that maybe it would be better to have everything in lower case inside `cookiecutter.quarto_theme`

and it would be even simpler:

```yaml
    theme: {{ cookiecutter.quarto_theme }}
```